app/src/main/java/ar/com/wolox/android/cookbook/datasync/Pokemon.kt

```diff
@@ -0,0 +1,22 @@
+package ar.com.wolox.android.cookbook.datasync
+
+data class Pokemon(
+    val id: Int,
+    val name: String,
+    private val sprites: Map<String, String>,
+    private val types: List<TypeSlot>
+) {
+
+    val spriteUrls
+        get() = sprites.values.toList()
+
+    val firstType: Type
+        get() = types.sortedBy { it.slot }[0].type
```

Comment:
Could not this be abstracted in another method like `getSortedTypeAt(type: Int)` for not repeat code?